xml/System.Windows.Forms/ListView.xml

```diff
@@ -2485,9 +2485,22 @@
         <ReturnType>System.Windows.Forms.ImageList</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>The currently set GroupIcon image list.</summary>
+        <value>An <see cref="T:System.Windows.Forms.ImageList" /> that contains the icons to use for <see cref="T:System.Windows.Forms.ListViewGroup" />.
+             The default is <see langword="null" />.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+ The <xref:System.Windows.Forms.ListView.GroupImageList> property allows you to specify an <xref:System.Windows.Forms.ImageList> object that contains icons to use when displaying groups. The <xref:System.Windows.Forms.ListView> control can accept any graphics format that the <xref:System.Windows.Forms.ImageList> control supports when displaying icons. The <xref:System.Windows.Forms.ListView> control is not limited to .ico files. Once an <xref:System.Windows.Forms.ImageList> is assigned to the <xref:System.Windows.Forms.ListView.GroupImageList> property, you can set the <xref:System.Windows.Forms.ListViewGroup.TitleImageIndex> property of each <xref:System.Windows.Forms.ListViewGroup> in the <xref:System.Windows.Forms.ListView> control to the index position of the appropriate image in the <xref:System.Windows.Forms.ImageList>.
+              The size of the icons for the <xref:System.Windows.Forms.ListView.GroupImageList> is specified by the <xref:System.Windows.Forms.ImageList.ImageSize> property.
+             
+
+
+
```

Comment:
Merging 4 empty lines into one:
```suggestion
             
```

Reply:
The remarks block on this method looks like this:

```
        /// <remarks>
        ///  <para>
        ///   The <see cref="GroupImageList"/> property allows you to specify an <see cref="ImageList"/> object that
        ///   contains icons to use when displaying groups. The <see cref="ListView"/> control can accept any graphics
        ///   format that the <see cref="ImageList"/> control supports when displaying icons. The <see cref="ListView"/>
        ///   control is not limited to .ico files. Once an <see cref="ImageList"/> is assigned to the <see cref="GroupImageList"/>
        ///   property, you can set the <see cref="ListViewGroup.TitleImageIndex"/> property of each <see cref="ListViewGroup"/>
        ///   in the <see cref="ListView"/> control to the index position of the appropriate image in the <see cref="ImageList"/>.
        ///   The size of the icons for the <see cref="GroupImageList"/> is specified by the <see cref="ImageList.ImageSize"/> property.
        ///  </para>
        /// </remarks>
```

I'm not sure where these empty lines are coming from